main.js

```diff
@@ -0,0 +1,243 @@
+//  ----------------VARIABLES--------------------------------//
+
+const nav = document.querySelector('.menu');
+const menubutton = document.querySelectorAll('#menubutton');
+const iconsButtons = document.querySelectorAll('.menu-links');
+const closeButtom = document.querySelectorAll('.close-buttom');
+
+//  -----------------Hamburger menu--------------------------//
+
+function toggleMenu() {
+  nav.classList.toggle('active');
+}
+
+function closeMenu() {
+  nav.classList.remove('active');
+}
+
+menubutton.onclick = () => { toggleMenu(); };
+iconsButtons.onclick = () => { closeMenu(); };
+
+//  -------------------Modal functions OPEN/CLOSE-----------------//
+
+function toggleModal(ID) {
+  const modal = document.getElementById(ID);
+  modal.classList.add('showUp');
+
+  window.onclick = function (event) {
+    if (event.target === modal || event.target.className === 'close-buttom') {
+      modal.classList.remove('showUp');
+    }
+  };
+
+  closeButtom.onclick = function () {
+    modal.classList.remove('showUp');
+  };
+}
+
+//  -------------------DATA STORAGE (array of objects)------------------------//
+
+//  CARDS DATA
+const projects = [
+  {
+    projectID: 0,
+    name: 'Multi-Post Stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    featuredImageBig: 'images/firstprojectbig.png',
+    featuredImageSmall: 'images/backgroundprojects2.png',
+    technologies: ['html', 'bootstrap', 'Ruby'],
+    linkLiveVersion: 'https://www.google.com',
+    linkSource: 'https://www.github.com',
```

Comment:
**[REQUIRED CHANGES]**

- Hey, please kindly ensure that the links to  **all live sources** take a user to the Github repository of the project specified in the live demo link.
**This change applies to all the project cards**

Reply:
✔️